Bindings/Java/tests/TestEditProperties.java

```diff
@@ -38,6 +38,23 @@ public static void main(String[] args) {
                     System.exit(-1);
                 } 
         }
+		// Now edit a Vec6 Property (inertia)
+		AbstractProperty inertiaProp = bdy.getPropertyByName("inertia");
+		Vec6 oldInertia = new Vec6();
+		for(int i=0; i<6; i++){
```

Comment:
Thanks @tkuchida This block tests that we can read existing Vec6 properties read from file. I couldn't assert here since there could be roundoff errors.